MapboxNavigation/MapboxVoiceController.swift

```diff
@@ -50,12 +50,24 @@ open class MapboxVoiceController: RouteVoiceController, AVAudioPlayerDelegate {
         muteToken = NavigationSettings.shared.observe(\.voiceMuted) { [weak self] (settings, change) in
             if settings.voiceMuted {
                 self?.audioPlayer?.stop()
+             
+                guard let self = self else { return }
```

Comment:
@hastiranjkesh could we refactor this line ... to say, `guard let strongSelf = self else { return }`